api/FontFace.json

```diff
@@ -240,24 +240,40 @@
           }
         }
       },
-      "style": {
+      "load": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FontFace/style",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FontFace/load",
           "support": {
-            "chrome": {
-              "version_added": null
-            },
-            "chrome_android": {
-              "version_added": null
-            },
+            "chrome": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "35",
+                "version_removed": "45",
+                "partial_implementation": true,
+                "notes": "Starting in Chrome 45, the returned promise resolves with a <code>FontFace</code> object as required by the specification instead of void."
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "35",
+                "version_removed": "45",
+                "partial_implementation": true,
+                "notes": "Starting in Chrome 45, the returned promise resolves with a <code>FontFace</code> object as required by the specification instead of void."
```

Comment:
```suggestion
                "notes": "The returned promise resolves void instead of a <code>FontFace</code> object as required by the specification."
```